index.js

```diff
@@ -236,41 +253,43 @@ const image = {
   },
   getOriginal (req, res) {
     const matches = parsing.getImageParams(req);
-    log.log('info', "Requested original image " + matches.fileName + " in format " + matches.fileType);
-    if (parsing.supportedFileType(matches.fileType)) {
-      const file = `${config.get('originals_dir')}/${matches.fileName}`;
-      fs.exists(file, function (exists) {
-        if (!exists) {
-          return helpers.send404(res, file);
-        }
-        let headers = {
-          'Content-Type': parsing.supportedFileType(matches.fileType),
-          'Cache-Control': 'public',
-          'Etag': `${matches.fileName}_${matches.fileType}`,
-          'Expires': helpers.farFutureDate()
-        };
-        res.writeHead(200, headers);
-        fs.readFile(file, function (err, data) {
-          res.end(data);
-        });
-      });
+    log.log('info', `Requested original image ${matches.fileName} in format ${matches.fileType}`);
+    if (!parsing.supportedFileType(matches.fileType)) {
+      return helpers.set404(res, req.url);
     }
+    const file = `${config.get('originals_dir')}/${matches.fileName}`;
+    fs.exists(file, (exists) => {
```

Comment:
FYI: `fs.exists` is deprecated and replaced with [`fs.access`](https://nodejs.org/api/fs.html#fs_fs_access_path_mode_callback) (It works a bit different from exists though).